### PR TITLE
Feat: Revamp draggable row

### PR DESCRIPTION
**Pull request by pKorsholm**
Base implementation of a draggable table, without dropping working due to how rapidly it updates state

also updating camelcase on some svgs

files to review: 
- `src/components/templates/draggable-table.tsx`

### CODE REVIEW

File: src/components/templates/draggable-table.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      update(records, {
+        $splice: [
+          [dragIndex, 1],
+          [hoverIndex, 0, dragRecord],
+        ],
+      })
+    )
+  }
+
+  const debouncedMoveRow = useMemo(() => _.debounce(moveRow, 100), [])
```
Comment (srindom) [inline src/components/templates/draggable-table.tsx:66]:
```suggestion
  const debouncedMoveRow = useMemo(() => debounce(moveRow, 100), [])
```

**Review by srindom — APPROVED** [review]
Lodash is notoriously heavy so better to only import individual functions to allow proper tree-shaking.

Code looks good otherwise; I assume that it can be seen in action somewhere in a later PR?

**Review by zakariaelas — APPROVED** [review]
Nice work Philip! Looking forward to seeing this in action :star_struck:

**Comment by pKorsholm** [comment]
@zakariaelas @srindom I just pushed a storybook story with the draggable table 😄  feel free to give it a shot

**Pull request merged**